docs/src/content/docs/fr/guides/pages.mdx

```diff
@@ -0,0 +1,151 @@
+En outre, les projets Starlight bénéficient d'un accès complet aux [puissants outils de génération de pages d'Astro](https://docs.astro.build/fr/basics/astro-pages/).
+Ce guide montre comment fonctionne la génération de pages dans Starlight.
+
+## Pages de contenu
+
+### Formats de fichiers
+
+Starlight prend en charge la création de contenu en Markdown et MDX sans aucune configuration requise.
+Vous pouvez ajouter la prise en charge de Markdoc en installant l'intégration expérimentale [Astro Markdoc](https://docs.astro.build/fr/guides/integrations-guide/markdoc/).
+
+### Ajouter des pages
+
+Ajoutez de nouvelles pages à votre site en créant des fichiers `.md` ou `.mdx` dans `src/content/docs/`.
+Utilisez des sous-dossiers pour organiser vos fichiers et pour créer plusieurs segments de chemin.
+
+Par exemple, la structure de fichier suivante générera des pages à `exemple.com/hello-world` et `exemple.com/reference/faq` :
+
+import FileTree from '~/components/file-tree.astro';
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - hello-world.md
+      - reference/
+        - faq.md
+
+</FileTree>
+
+### Sécurité de la typologie du frontmatter
+
+Toutes les pages Starlight partagent un [ensemble commun de propriétés de la page d'accueil](/fr/reference/frontmatter/) personnalisable qui permet de contrôler l'apparence de la page :
+
+```md
+---
+title: Bonjour tout le monde !
+description: Voici une page de mon site alimenté par Starlight
+---
+```
+
+Si vous oubliez quelque chose d'important, Starlight vous le fera savoir.
+
+## Pages personnalisées
+
+Pour les cas d'utilisation avancés, vous pouvez ajouter des pages personnalisées en créant un répertoire `src/pages/`.
+Le répertoire `src/pages/` utilise le [routage basé sur les fichiers d'Astro](https://docs.astro.build/fr/basics/astro-pages/#file-based-routing) et inclut le support des fichiers `.astro` parmi d'autres formats de pages.
+Ceci est utile si vous avez besoin de construire des pages avec une mise en page complètement personnalisée ou de générer une page à partir d'une source de données alternative.
+
+Par exemple, ce projet mélange du contenu Markdown dans `src/content/docs/` avec des routes Astro et HTML dans `src/pages/` :
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - hello-world.md
+  - pages/
+    - custom.astro
+    - archived.html
+
+</FileTree>
+
+Pour en savoir plus, consultez le guide ["Pages" dans la documentation Astro](https://docs.astro.build/fr/basics/astro-pages/).
+
+### Utiliser le design de Starlight dans des pages personnalisées
+
+Pour utiliser la mise en page Starlight dans des pages personnalisées, englobez le contenu de votre page avec le composant `<StarlightPage />`.
+Cela peut s'avérer utile si vous générez du contenu de manière dynamique, mais que vous souhaitez tout de même utiliser la conception de Starlight.
```

Comment:
```suggestion
Cela peut s'avérer utile si vous générez du contenu de manière dynamique, mais que vous souhaitez tout de même utiliser le design de Starlight.
```

I think we should keep the consistency with the heading like the English version and "conception" does not feel right to me here.